doc/pr2_tutorials/planning/src/doc/moveit_commander.rst

```diff
@@ -0,0 +1,47 @@
+MoveIt! Commander and Command Line Tool
+=======================================
+
+The `moveit_commander <http://wiki.ros.org/moveit_commander>`_ Python package offers wrappers for the functionality provided in MoveIt!. Simple interfaces are available for motion planning, computation of Cartesian paths, and pick and place.
+
+Command Line Tool
+-----------------
+
+The **moveit_commander** package also includes a command line tool for controlling the robot. 
+
+First, launch the MoveIt! demo interface: ::
```

Comment:
This is only needed if you did not start a `move_group` node with your real robot,
but you can (of course) run the MoveIt commander with the real robot too.

Could you please clarify here that the "demo" mode is the simulated moveit environment?